responses/fr/HassMediaNext.yaml

```diff
@@ -0,0 +1,5 @@
+language: fr
+responses:
+  intents:
+    HassMediaNext:
+      default: "Suivant"
```

Comment:
I find "Suivant" weird but my only proposal is "Élement suivant" because we do not know if the media player is paying a movie, a song or something else...
And I also do not like it... 🙈

Reply:
what about "Média suivant"?

Reply:
I like it !